src/frontend/packages/lib_tests/src/cunningham.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen } from '@testing-library/react';
+import userEventInit from '@testing-library/user-event';
+import { DateTime } from 'luxon';
+
+/**
+ * To simulate a user typing a date in the Cunningham date picker
+ * @param startingAt
+ * @param userEvent
+ */
+export const userTypeDatePicker = async (
+  startingAt: DateTime,
+  userEvent?: ReturnType<typeof userEventInit.setup>,
+) => {
+  const localUserEvent = userEvent || userEventInit;
+  await localUserEvent.click(screen.getByText(/Starting date/i));
```

Comment:
screen.getByText(/Starting date/i) makes your function not generic, maybe we should give it an element through a parameter

Reply:
[fixup! 💄(frontend) replace to cunningham Datepicker](https://github.com/openfun/marsha/pull/2297/commits/ea3f0d67d4fde8d5f9b03bfe364ee5361204e014)